blog/20140317-galculator-coming-to-mate.de.md

```diff
@@ -1,5 +1,5 @@
 <!-- 
-.. link: http://galculator.sourceforge.net/
+.. link: https://github.com/galculator/galculator
```

Comment:
I think is better for galculator http://galculator.mnim.org/ with detailed info about program.

Reply:
I have chosen github because of https, anyway, i will change it.
I hope all that http links doesn't make a problem when i try to use our domain name as custom domain name for gh-pages for mate-wiki.